devicetypes/smartthings/neo-coolcam-light-switch.src/neo-coolcam-light-switch.groovy

```diff
@@ -0,0 +1,164 @@
+/**
+ *  Copyright 2018 SmartThings
+ *
+ *  Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License. You may obtain a copy of the License at:
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *  on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *  for the specific language governing permissions and limitations under the License.
+ *
+ */
+metadata {
+	definition(name: "NEO Coolcam Light Switch", namespace: "smartthings", author: "SmartThings") {
+		capability "Actuator"
+		capability "Health Check"
+		capability "Light"
+		capability "Refresh"
+		capability "Sensor"
+		capability "Switch"
+
+		fingerprint mfr: "0258", prod: "0003", model: "008B", deviceJoinName: "NEO Coolcam Light Switch"
+	}
+
+	// tile definitions
+	tiles(scale: 2) {
+		multiAttributeTile(name: "switch", type: "lighting", width: 6, height: 4, canChangeIcon: true) {
+			tileAttribute("device.switch", key: "PRIMARY_CONTROL") {
+				attributeState "on", label: '${name}', action: "switch.off", icon: "st.switches.switch.on", backgroundColor: "#00A0DC"
+				attributeState "off", label: '${name}', action: "switch.on", icon: "st.switches.switch.off", backgroundColor: "#ffffff"
+			}
+		}
+
+		standardTile("refresh", "device.switch", width: 2, height: 2, inactiveLabel: false, decoration: "flat") {
+			state "default", label: '', action: "refresh.refresh", icon: "st.secondary.refresh"
+		}
+
+		main "switch"
+		details(["switch", "refresh"])
+	}
+}
+
+def installed() {
+	def componentLabel = "NEO Coolcam Light Switch ep2"
```

Comment:
"ep2" isn't a very user-friendly term for the label. How about just "NEO Coolcam Light Switch 2"? And maybe use ".. Switch 1" for the deviceJoinName of the main device?